scenariolib/config.go

```diff
@@ -101,22 +159,22 @@ func (c *Config) RandomQueryInLanguage(good bool, language string) (string, erro
 
 // RandomUserAgent returns a random user agent string to send with an event
 func (c *Config) RandomUserAgent(mobile bool) (string, error) {
```

Comment:
This func uses only `RandomData`, why not pass it a `c *RandomData` instead?

Reply:
This was such a good idea 👯‍♂️